app/app.go

```diff
@@ -466,6 +560,9 @@ func New(
 	ibcRouter.AddRoute(ibctransfertypes.ModuleName, transferStack)
 	app.IBCKeeper.SetRouter(ibcRouter)
 
+	app.EvmKeeper.WithStaticPrecompiles(
```

Comment:
I think the sdk50 upgarde is big enough already, we should not add precompiles yet, that should probably be on an independent updrage.